src/Injector/inject.cpp

```diff
@@ -0,0 +1,41 @@
+#include <iostream>
+#include <Windows.h>
+
+int inject(DWORD PID)
+{
+    // path to dll
+    //LPCSTR DllPath = "../../Hooks/debug/Hooks.dll";
+    LPCSTR DllPath = "C:\\CodeBase\\Apriorit-Project\\build-src-Desktop_Qt_6_3_1_MinGW_64_bit-Debug\\Hooks\\debug\\Hooks.dll";
```

Comment:
I understood what you meant when told about absolute path. There should be no hardcode at all. This path should be passed to the function somehow

Reply:
done.